woocommerce/woocommerce
# Exclude marketing channels in "Installed extensions" and "Discover more marketing tools" cards

### All Submissions:


### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/34907.

In this PR, we filter out marketing channels in the "Installed extensions" and "Discover more marketing tools" cards. This is because marketing channels already show up in its own "Channels" card.

As a result, there should be no more "Sales channels" tab in the "Discover more marketing tools" card. Each extension should only appear in one card.

Screenshot:

![image](https://user-images.githubusercontent.com/417342/223767871-5a2133e3-ac2c-4178-944b-4955176faa9c.png)

### How to test the changes in this Pull Request:




Pre-condition: 

- Enable WooCommerce Multichannel Marketing experience in WooCommerce Settings > Advanced > Features (`/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`).

Test instructions:

1. Go to Marketing page at `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`.
2. Each extension should only appear in one card.
3. There should be no "Sales channel" tab in the "Discover more marketing tools" card.



### Other information:




### FOR PR REVIEWER ONLY:
